app/utils/article-link.js

```diff
@@ -61,9 +62,9 @@ export default function getLinkInfo(basePath, title, hash, uri, queryString) {
 
```

Comment:
It's not shown in the diff but links to the main page might be broken, I think. They lack the `langPath` part.

Reply:
Fixed in https://github.com/Wikia/mobile-wiki/pull/618/commits/e8d26be07056d57729957f528ae7285e6212f422